content/en/continuous_integration/pipelines/jenkins.md

```diff
@@ -1513,3 +1525,5 @@ Failed to reinitialize Datadog-Plugin Tracer, Cannot enable traces collection vi
 [27]: /logs/guide/best-practices-for-log-management/
 [28]: /continuous_integration/search/#search-for-pipelines
 [29]: /agent/logs/?tab=tcpudp#custom-log-collection
+[30]: /developers/dogstatsd/
+[31]: https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml
```

Comment:
```suggestion
[31]: /containers/docker/apm/#tracing-from-the-host
```
I think this config template link might be a little confusing for users. How do you feel about this link instead?

Reply:
Agree about the template being confusing. I couldn't find a suitable link, but this one seems like a good fit. Thanks!